mucommander-viewer-image/src/main/java/com/mucommander/viewer/image/ImageViewer.java

```diff
@@ -110,86 +160,121 @@ public void setPresenter(ViewerPresenter presenter) {
 
     @Override
     public void extendMenu(JMenuBar menuBar) {
+        menuBar.add(viewMenu);
         menuBar.add(controlsMenu);
     }
 
     private synchronized void loadImage(AbstractFile file) throws IOException {
-        presenter.getWindowFrame().setCursor(new Cursor(Cursor.WAIT_CURSOR));
-		
+        presenter.getWindowFrame().setCursor(Cursor.getPredefinedCursor(Cursor.WAIT_CURSOR));
+
         int read;
         byte buffer[] = new byte[1024];
-        ByteArrayOutputStream bout = new ByteArrayOutputStream();
-        InputStream in = file.getInputStream();
-        while ((read=in.read(buffer, 0, buffer.length))!=-1)
-            bout.write(buffer, 0, read);
-
-        byte imageBytes[] = bout.toByteArray();
-        bout.close();
+        InputStream in;
+        byte[] imageBytes;
+        try (ByteArrayOutputStream bout = new ByteArrayOutputStream()) {
+            in = file.getInputStream();
```

Comment:
how about adding `in` to the try-with-resources statement above
```java
try (ByteArrayOutputStream bout = new ByteArrayOutputStream();
     InputStream in =  file.getInputStream()) {
  ....
```

Reply:
Replaced by StreamUtis.copyStream